trdg/data_generator.py

```diff
@@ -174,6 +174,26 @@ def generate(
             "RGB", (background_width, background_height), (0, 0, 0)
         )
 
+        ##############################################################
+        # Comparing average pixel value of text and background image #
+        ##############################################################
+        try:
+            resized_img_st = ImageStat.Stat(resized_img, resized_mask.split()[2])
+            background_img_st = ImageStat.Stat(background_img) 
+
+            resized_img_px_mean = sum(resized_img_st.mean[:2]) / 3
+            background_img_px_mean = sum(background_img_st.mean) / 3
+
+            if abs(resized_img_px_mean - background_img_px_mean) < 15:
+                print("value of mean pixel is too similar. Ignore this image")
+
+                print("resized_img_st \n {}".format(resized_img_st.mean))
+                print("background_img_st \n {}".format(background_img_st.mean))
+
+                return
+        except Exception as err:
+            return
```

Comment:
Is this `return` needed? For the `ZeroDivisionError` which can occur in line 184 it would be better not to return. That would fix the regression for `-hw`.